solvebio/global_search.py

```diff
@@ -185,14 +185,14 @@ def subjects(self):
         """If entity seaarch is performed returns the list of subjects"""
 
         # Executes a query to get a full API response which contains subjects list
-        self.execute()
+        self.execute(include_subjects=True, limit=0)
 
         return self._response.get('subjects')
 
     def subjects_count(self):
         """If entity seaarch is performed returns the list of subjects"""
 
         # Executes a query to get a full API response which contains subjects list
-        self.execute()
+        self.execute(include_subjects=True, limit=0)
```

Comment:
subjects_count is actually always included, so you can set include_subjects=False here

Reply:
Maybe I'm wrong but haven't you mentioned on the call that currently subjects are always included, but that we're going to change the API so by default it doesn't return subjects all the time, to make it faster?

Reply:
Yes, but the subject count field is always returned regardless.

Reply:
Aha ok ok, understood.